packages/web-components/src/accordion-item/accordion-item.ts

```diff
@@ -101,46 +101,59 @@ export class AccordionItem extends FASTElement {
   public id: string = uniqueId('accordion-');
 
   /**
-   * Defines accordion header font size.
+   * Sets the width of the focus state.
    *
    * @public
    * @remarks
-   * HTML Attribute: size
+   * HTML Attribute: block
    */
-  @attr
-  public size?: AccordionItemSize;
+  @attr({ mode: 'boolean' })
+  public block: boolean = false;
```

Comment:
This may be presentational per some mockups of accordions I've seen recently.

Reply:
yup, I'll move.